addons/versioning/XEH_preInit.sqf

```diff
@@ -46,6 +46,11 @@ PREP(version_check);
 FUNC(version_compare) = {
     private ["_failed", "_c"];
     params ["_value","_localValue"];
+
+    //Handle non-number arrays - eg. {"mod", {"1.0"}, "(true)"}
+    if ((!(_value isEqualTo [])) && {!(_value isEqualTypeAll 0)}) exitWith {true};
+    if ((!(_localValue isEqualTo [])) && {!(_localValue isEqualTypeAll 0)}) exitWith {true};

```

Comment:
Except it's not the dependency that is bad, it's the configuration that is incorrect.